keras/layers/convolutional.py

```diff
@@ -1229,25 +1230,6 @@ def call(self, inputs):
             return self.activation(outputs)
         return outputs
 
-    def compute_output_shape(self, input_shape):
```

Comment:
Can you comment on why you are removing this method? The parent already works?

Reply:
Yeah, the parent method is doing the same thing as this one, but it computes the output shape with the dilation rate.